backend/src/main/java/com/daedan/festabook/global/logging/LoggingFilter.java

```diff
@@ -17,15 +18,29 @@
 @Component
 public class LoggingFilter extends OncePerRequestFilter {
 
+    private static final List<String> LOGGING_SKIP_PATH_PREFIX = List.of(
+            "/api/swagger-ui/",
+            "/api/v3/api-docs/",
+            "/api/api-docs",
+            "/api/actuator/"
+
```

Comment:
1. 26번 라인에 불필요한 공백이 있는 걸로 보여요!

2. prefix 라서 전부

```suggestion
            "/api/swagger-ui",
            "/api/v3/api-docs",
            "/api/api-docs",
            "/api/actuator"
```

로 시작해도 될 거 같아요!